src/BenchmarksApps/Kestrel/PlatformBenchmarks/Data/RawDb.cs

```diff
@@ -237,7 +259,55 @@ private async Task<World> ReadSingleRow(NpgsqlCommand cmd)
             }
         }
 
-        private static readonly object[] _cacheKeys = Enumerable.Range(0, 10001).Select((i) => new CacheKey(i)).ToArray();
+        internal class SqlFortuneCommand : IDisposable
```

Comment:
Are these wrapper classes actually necessary, why not pool NpgsqlCommand directly? As far as I can tell they're mainly there to enqueue back when disposing, but that can just be done by the code using the command instead of disposing, no?

Reply:
Mainly so the callsites are just a `using` block and doon't have to worry about pooling e.g. its
```csharp
using (var cmd = CreateReadCommand())
{
    cmd.Connection = db;
    // do something with pooled command
}
```

Reply:
OK. Moving the pooling logic here and removing the wrappers might make a tiny bit of difference too.